examples/uniswap-v2-intents-goerli/.env.example

```diff
@@ -0,0 +1,2 @@
+GOERLI_KEY=
```

Comment:
should we have a .env per project or for the whole repo?

Reply:
whole-project .env is the simplest solution, especially as we add more example projects. moved it up to the project root in c066fe3